index.html

```diff
@@ -206,36 +206,41 @@ <h3>
         &lt;button&gt;Verify Identity&lt;/button&gt;
         &lt;script&gt;
           const button = document.querySelector("button");
-          button.addEventListener("click", async () =&gt; {
-            try {
-              const credential = await navigator.credentials.get({
-                digital: {
-                  requests: [{
-                    protocol: "example-protocol",
-                    data: { /* request data */ }
-                  }]
+          button.addEventListener("click", async () => {
+            // Check for DC API and protocol support
+            if (DigitalCredential?.userAgentAllowsProtocol("example-protocol")) {
```

Comment:
My preference is to just short circuit here to avoid nesting. 

```suggestion
            if (DigitalCredential?.userAgentAllowsProtocol("example-protocol") === false) {
                return;
            }
```